code/UMLModel.py

```diff
@@ -426,11 +403,8 @@ def list_relationships(self, class_name:str = ""):
                 if not self.classes[class_name].relationships:
                     print("Class '" + class_name + "' has no relationships")
                 else:
-                    for relationship in self.classes[class_name].relationships:
-                        if relationship.class1.name == class_name:
-                            print (class_name,"---", relationship.name, "-->",relationship.class2.name)
-                        else: 
-                            print (class_name,"---", relationship.name, "-->",relationship.class1.name)
+                    for other, relationship in self.classes[class_name].relationships.items():
+                        print (class_name,"---", relationship.name, "-->",relationship.get_other_class(class_name).name)
```

Comment:
with get_other_class(), you do not need to determine which class is which in the relationship object.